test/OpenTelemetry.Exporter.Geneva.Benchmark/Exporter/MetricExporterBenchmarks.cs

```diff
@@ -581,7 +581,7 @@ public void SerializeCounterMetricItemWith3Dimensions()
             this.counterMetricPointWith3Dimensions.Tags,
             this.counterMetricDataWith3Dimensions,
 #if EXPOSE_EXPERIMENTAL_FEATURES
-            Array.Empty<Exemplar>(),
+            default,
```

Comment:
This would throw an exception in the enumerator of `ReadOnlyExemplarCollection` since `Exemplar[]` would be `null` by default.

We should update the TlvMetricExporter code to call `SerializeExemplars` only when `ReadOnlyExemplarCollection` is not default.

Reply:
`exemplars != default` is not possible rn. I used a workaround here.